src/main/java/net/eventsexpress/app/Pages/AdminPanel.java

```diff
@@ -0,0 +1,37 @@
+package net.eventsexpress.app.Pages;
+
+import java.util.List;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+
+import net.eventsexpress.app.Common.BaseWrapper;
+
+public class AdminPanel extends BaseWrapper{
```

Comment:
I would recommend to place 'Page' after all page objects. It makes code more clear.